gitpod.yml

```diff
@@ -0,0 +1,3 @@
+vscode:
```

Comment:
The file has to be called .gitpod.yml https://www.gitpod.io/docs/config-gitpod-file